Reporting/testingPlan.md

```diff
@@ -1,6 +1,27 @@
-# Testing Approaches
+# Software Development Process and Testing Methodology
+
+We will follow a Test-Driven Development (TDD) approach whereby initial test cases are written before application components are developed. As our database is central to our feature development, test cases rely on mock user data supplied to our MongoDB Atlas database on the cloud. As we ease into the development of application components, we will proceed with black-box testing as we begin to finalize components.
+
+# Software Testing Approaches
```

Comment:
Nicely done with the detail on our approaches to using unit-testing for the back-end methods and black-box testing for our components.